pro/src/pages/AdageIframe/app/components/OffersInstantSearch/OffersSearch/Offers/OfferDetails/OfferDetails.tsx

```diff
@@ -89,6 +93,12 @@ const OfferDetails = ({
         </OfferSection>
       )}
 
+      {nationalProgram?.name && isNationalProgramActive && (
```

Comment:
Micro détail, mais j'aurais changé l'ordre des conditions. Si le FF n'est pas activé, le reste de la condition ne sera pas vérifié.

```suggestion
      {isNationalProgramActive && nationalProgram?.name && (
```

Reply:
Oui bonne idée, merci!